ios/l2_interfaces/ios_l2_interface.yaml

```diff
@@ -0,0 +1,74 @@
+---
+GENERATOR_VERSION: '1.0'
+
+ANSIBLE_METADATA: |
+    {
+      'metadata_version': '1.1',
+      'status': ['preview'],
+      'supported_by': 'network'
+    }
+NETWORK_OS: ios
+RESOURCE: l2_interfaces
+COPYRIGHT: Copyright 2019 Red Hat
+
+DOCUMENTATION: |
+  module: ios_l2_interfaces
+  version_added: 2.9
+  short_description: Manage Layer-2 interface on Cisco IOS devices.
+  description: This module provides declarative management of Layer-2 interface on Cisco IOS devices.
+  author: Sumit Jaiswal (@justjais)
+  options:
+    config:
+      description: A dictionary of Layer-2 interface options
+      type: list
+      elements: dict
+      suboptions:
+        name:
+          description:
+          - Full name of interface, e.g. ge-0/0/0.
+          type: str
+          required: True
+        mode:
+          default: access
+          description:
+          - Mode in which interface needs to be configured
+          type: str
+          choices: ['access', 'trunk']
+        access_vlan:
+          description:
+          - Configure given VLAN in access port. If C(mode=access), used as the access VLAN ID.
+        type: str
+        trunk_vlans:
+          description:
+          - List of VLANs to be configured in trunk port. If C(mode=trunk), used as the VLAN
+            range to ADD or REMOVE from the trunk.
+          type: str
+        native_vlan:
+          description:
+          - Native VLAN to be configured in trunk port. If C(mode=trunk), used as the trunk
+            native VLAN ID.
+          type: str
+        trunk_allowed_vlans:
+          description:
+          - List of allowed VLANs in a given trunk port. If C(mode=trunk), these are the only
+            VLANs that will be configured on the trunk, i.e. "2-10,15".
+          type: str
+        aggregate:
```

Comment:
Can't this be removed?  (looks like a carry over from an old module)

Reply:
removed